server/pkg/git/run.go

```
@@ -0,0 +1,338 @@
+}
+
+func hasUnmerged(state SiteState) bool {
+	for _, file := range state.Files {
+		if file.Unmerged {
+			return true
+		}
+	}
+	return false
+}
+
+func hasUntracked(state SiteState) bool {
```

golangci-lint (2.12.2)

[error] func hasUntracked is unused

(unused)